packages/binding.core/spec/hasfocusBehaviors.ts

```diff
@@ -42,6 +44,33 @@ arrayForEach(['hasfocus', 'hasFocus'], binding => {
       beforeEach(function () { waits(100) })
     }
 
+    it('Should set an observable value to be true on focus and false on blur even if the binding is applied through another binding', function () {
+      const createElementWithHasFocusBinding  = {
+        init: (element: HTMLElement, valueAccessor: () => any) => {
+          let parent = $(element);
+          let $hasFocus = valueAccessor();
+
+          applyBindingsToNode(parent[0], { hasFocus: $hasFocus })
+        }
+      };
+      options.bindingProviderInstance.bindingHandlers.set("customBinding",  createElementWithHasFocusBinding)
+
+      testNode.innerHTML = `<input data-bind='customBinding: customProps' /><input />`
+
+      var $myVal = observable(false)
```

Comment:
prefer `const`.  The `$` prefix is generally reserved for "private" properties/accessors.  It's ok here, just noting.